### PR TITLE
Speed up get-data-cost function

### PR DESCRIPTION
**Pull request by jdavis103**
#### Problem
Code that computes costs of transactions is quite slow.

#### Summary of Changes
Combine two cost-computation functions (that each loop through instructions) into a single function.

Fixes #

### CODE REVIEW

File: runtime/src/cost_model.rs [hunk 1]
```diff
@@ -177,8 +166,9 @@ impl CostModel {
                 );
                 bpf_costs = bpf_costs.saturating_add(instruction_cost);
             }
+            data_bytes_costs = data_bytes_costs.saturating_add(instruction.data.len() as u64);
```
Comment (taozhu-chicago) [inline runtime/src/cost_model.rs:169]: nit: `data_bytes_cost` is misleading, it is `data_bytes_len`

File: runtime/src/cost_model.rs [hunk 2]
```diff
@@ -149,20 +148,10 @@ impl CostModel {
...
-            .message()
-            .program_instructions_iter()
-            .for_each(|(_, ix)| {
-                data_bytes_cost += ix.data.len() as u64 / DATA_BYTES_UNITS;
-            });
-        data_bytes_cost
-    }
-
-    fn get_transaction_cost(&self, transaction: &SanitizedTransaction) -> (u64, u64) {
+    fn get_transaction_cost(&self, transaction: &SanitizedTransaction) -> (u64, u64, u64) {
```
Comment (taozhu-chicago) [inline runtime/src/cost_model.rs:151]: nit: maybe return a `struct` with named fields in place of a tuple

**Review by taozhu-chicago — COMMENTED** [review]
Good idea of reducing dup iteration. few nit picks

**Review by taozhu-chicago — APPROVED** [review]
lgtm

**Review by CriesofCarrots — APPROVED** [review]
Looks nice to me

**Pull request merged**